src/components/ContributorTasks/ContributorTasks.test.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import format from 'date-fns/format';
+
+import ContributorTasks, {ContributorTasksProps} from './index';
+
+describe('ContributorTasks', () => {
+  const props: ContributorTasksProps = {
```

Comment:
you can move this `props` outside of the `describe`. Put it in between the last `import` (on line 7) and before the `describe`